resources/views/frontend/trainee-profile/edit-personal-info.blade.php

```diff
@@ -20,174 +56,174 @@
+                                                    value="{{ \App\Models\TraineeFamilyMemberInfo::PHYSICALLY_DISABLE_YES }}"
+                                                    {{ $trainee->disable_status == \App\Models\TraineeFamilyMemberInfo::PHYSICALLY_DISABLE_YES ? 'checked' : '' }}>
+                                                <label for="physically_disable_yes"
+                                                    class="custom-control-label">{{ __('generic.yes') }}</label>
+                                            </div>
+                                            <div class="custom-control custom-radio mr-3">
+                                                <input class="custom-control-input" type="radio"
+                                                    id="physically_disable_no" name="disable_status"
+                                                    value="{{ \App\Models\TraineeFamilyMemberInfo::PHYSICALLY_DISABLE_NOT }}"
+                                                    {{ $trainee->disable_status == \App\Models\TraineeFamilyMemberInfo::PHYSICALLY_DISABLE_NOT ? 'checked' : '' }}>
+                                                <label for="physically_disable_no"
+                                                    class="custom-control-label">{{ __('generic.no') }}</label>
+                                            </div>
                                         </div>
                                     </div>
-                                </div>
 
-                                <div class="form-group col-md-12">
-                                    <label for="address">{{ __('generic.address') }}</label>
-                                    <textarea class="form-control" name="address" placeholder="Address"
-                                              id="address">{{ old('address', $trainee->address) }}</textarea>
-                                </div>
+                                    <div class="form-group col-md-12">
+                                        <label for="address">{{ __('generic.address') }}</label>
+                                        <textarea class="form-control" name="address" placeholder="Address" id="address">{{ old('address', $trainee->address) }}</textarea>
+                                    </div>
 
-                                <div class="form-group col-md-6">
-                                    <label
-                                        for="password">{{ __('generic.old_password') }}</label>
-                                    <input type="password" class="form-control" name="old_password"
-                                           id="old_password"
-                                           placeholder="{{ __('generic.old_password') }}"
-                                    >
-                                </div>
+                                    <div class="col-md-12">
+                                        <input type="submit"
```

Comment:
```
<input type="submit"
                                            class="btn btn-sm btn-primary float-right px-3 pt-2 pb-1 custom-button-style"
                                            value="{{-- {{ __('admin.common.edit') }} --}} Save Changes">
```
Just copy this code. I have fixed the spelling as well as stying issues for alignment that you missed.